src/ProjectOrigin.WalletSystem.IntegrationTests/Repositories/CertificateRepositoryTests.cs

```diff
@@ -174,6 +174,85 @@ public async Task GetAllOwnedCertificates()
         );
     }
 
+    [Fact]
+    public async Task GetAllOwnedCertificates_WhenTypeIsConsumption()
+    {
+        // Arrange
+        var endpointPosition = 1;
+        var registry = _fixture.Create<string>();
+        var certificate1 = await CreateCertificate(registry);
+        var certificate2 = await CreateCertificate(registry, GranularCertificateType.Consumption);
+        var certificate3 = await CreateCertificate(registry);
+        var owner1 = _fixture.Create<string>();
+        var wallet1 = await CreateWallet(owner1);
+        var endpoint1 = await CreateWalletEndpoint(wallet1);
+        var endpoint2 = await CreateWalletEndpoint(wallet1);
+        var owner2 = _fixture.Create<string>();
+        var wallet2 = await CreateWallet(owner2);
+        var endpoint3 = await CreateWalletEndpoint(wallet2);
+        //Wallet1
```

Comment:
Not sure this comment makes sense. Is it not "certificate 1"? And there is no comment for sliceWithDifferentOwner

Reply:
@tnickelsen would you comment this?

Reply:
i rewrote it a bit for clarity... i hope

Reply:
It is easier to read now